docs/source/tutorials/tiled_ensemble.rst

```diff
@@ -0,0 +1,125 @@
+.. _tiled_ensemble:
+
+Tiled ensemble
+--------------
+Anomalib provides tiled ensemble training. Images are split into tiles and separate model is trained for each tile location.
+
+
+.. image:: ../images/tiled_ensemble/design.png
+    :alt: Tiled ensemble design
+
+Training
+========
+
+Training of tiled ensemble can be done with training script using the following command:
+
+.. code-block:: bash
+
+    python tools/tiled_ensemble/train_ensemble.py \
+        --ens_config tools/tiled_ensemble/ens_config.yaml 
+
+By default padim model is trained on bottle category. Another model can be passed like in following example:
+
+.. code-block:: bash
+
+    python tools/tiled_ensemble/train_ensemble.py \
+        --model fastflow \
+        --ens_config tools/tiled_ensemble/ens_config.yaml 
+
+Or using a path to config:
+
+.. code-block:: bash
+
+    python tools/tiled_ensemble/train_ensemble.py \
+        --config src/anomalib/models/padim/config.yaml \
+        --ens_config tools/tiled_ensemble/ens_config.yaml 
```

Comment:
Would a user find it hard to follow the difference between `config` and `ens_config`? Would it be an idea to explicitly define these such as:

```bash
    python tools/tiled_ensemble/train_ensemble.py \
         --model_config src/anomalib/models/padim/config.yaml \
         --ensemble_config tools/tiled_ensemble/ens_config.yaml
```

Reply:
Yes, a very valid concern. I think it is confusing and renaming would be really needed. I will change to model_config and ensemble_config.

Reply:
Done